en/components/themes/examples.md

```diff
@@ -277,7 +277,16 @@ The component schemas are applied through the global schemas, in order to use th
         )
     );
 ```
-Finally you have to define the `$palette` and set the `$schema` property of the components themes with the new global schema, defined above.
+Finally, you have to define the `$palette`:
+```scss
+...
+$black-color: #000000;
+$aqua-color: #00BFFF;
+
+$black-theme-palette: igx-palette($primary: $black-color, $secondary: $aqua-color);
```

Comment:
Make sure you set the $grays property to #fff. This is important when creating palettes that are gonna be used with dark schemas.

Reply:
Added $grays parameter with and a note for the readers to apply this when they extend the $dark-schema